docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -27,7 +27,7 @@ The main media controller features include:
 
  - Media Control Metadata Properties
```

Comment:
Global comment:
- Updating and retrieving information
- Updating and Retrieving Playlist
- Sending and processing commands
- Setting and Getting Playback Capability
- Media Control Metadata Properties

There is an inconsistency in the list, some are in sentence case while others are in the title case. To maintain consistency, should we follow any one style?

@mijin85cho 
Please verify and share your opinion regarding this.

Reply:
I'll change it to `Setting and Getting Media Control Metadata`.

Reply:
Please wait for @mijin85cho suggestion before updating.

Reply:
When there's no statement and code for `setting and getting metadata`(172~186), `Media Control Metadata was also good. 
But it has some statement and code for getting and setting metadata now, so I think your suggestion is not wrong.